worlds/rogue_legacy/Regions.py

```diff
@@ -56,9 +59,9 @@ def create_regions(multiworld: MultiWorld, player: int):
     regions["The Fountain Room"].locations.append("Fountain Room")
 
     # Chests
-    chests = int(multiworld.chests_per_zone[player])
+    chests = int(world.options.chests_per_zone.value)
```

Comment:
Here and elsewhere, with `int()` you don't need `.value` for Range options
```suggestion
    chests = int(world.options.chests_per_zone)
```